src/azure-cli/azure/cli/command_modules/backup/tests/latest/test_workload_commands.py

```diff
@@ -561,13 +561,144 @@ def test_backup_wl_sql_restore_as_files(self):
 
         self.cmd('backup job wait -v {vault} -g {rg} -n {job}')
 
+    @record_only()
```

Comment:
May I ask why those tests are `@record_only()`?

Reply:
all workload tests have @record_only tag on them. So, I felt it would be prudent to be in line with the pattern that has been followed till now.

Reply:
Only when the test can't run in live mode due to some problems, we mark it as `@record_only`. Since we have a pipeline running live tests regularly to ensure the test passing rate. If you don't need to mark it as `@record_only`, please try not to mark it

Reply:
Hey Xing. We are working on creating a separate test setup exclusively for workload tests. Till then we are using existing machines to run tests. Sometimes it may lead to the tests getting failed in live mode. Please let us use @record_only tag for workload tests till the test setup is ready.

Reply:
OK